deploy/Managing_Keystores_And_Truststores.md

```diff
@@ -0,0 +1,71 @@
+# Managing Java Keystores and Truststores
+
+This chapter depicts important information about managing Java keystores and truststores in a container based
+WSO2 product deployment.
+
+## Contents
+
+The following will be discussed in detail in the document.
+
+* [Asymmetric Encryption in WSO2 products](#asymmetric-encryption-in-wso2-products)
+* [Manage Keystores and Certificate Truststores in Container Based Deployments](#manage-keystores-and-certificate-truststores-in-container-based-deployments)
+
+### Asymmetric Encryption in WSO2 Products
+
+WSO2 products use asymmetric encryption by default for the authentication and protection of data. For advanced details,
+please refer the official WSO2 [documentation](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-setting_up_keystoresUsageofkeystoresinWSO2products).
+
+During this process, [Java keystores and truststores](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-Understandingkeystoresandtruststores)
+are used.
+
+Please refer the official WSO2 [recommendations](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-recommendationsRecommendationsforsettingupkeystoresinWSO2products)
+for managing keystores prior to following this guide.
+
+### Manage Keystores and Certificate Truststores in Container Based Deployments
+
+As per aforementioned WSO2 [recommendations](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption#UsingAsymmetricEncryption-recommendationsRecommendationsforsettingupkeystoresinWSO2products),
+it is often required to customize the default, WSO2 product keystore and truststore in a production grade deployment.
+
+The most appropriate approach for introducing a custom keystore or a truststore to a container based deployment is via
+volume mounting.
+
+1. Prepare the custom Java keystore.
+
+   Please follow the official guidelines for [creating a new keystore](https://docs.wso2.com/display/ADMIN44x/Creating+New+Keystores#CreatingNewKeystores-Creatinganewkeystore)
+   or [adding a Certificate Authority signed certificate to an existing keystore](https://docs.wso2.com/display/ADMIN44x/Creating+New+Keystores#CreatingNewKeystores-ca_certificateAddingCA-signedcertificatestokeystores)
+   as per your preference.
+   
+   Also, please follow the official [documentation](https://docs.wso2.com/display/ADMIN44x/Creating+New+Keystores#CreatingNewKeystores-Step3:Importingcertificatestothetruststore)
```

Comment:
To import certificates to a truststore follow this [documentation](https://docs.wso2.com/display/ADMIN44x/Creating+New+Keystores#CreatingNewKeystores-Step3:Importingcertificatestothetruststore)